table_operations/rasgo_union/rasgo_union.yaml

```diff
@@ -0,0 +1,7 @@
+name: union
```

Comment:
in the yamls let's use the standard opening and closing syntax of the two dashes so its:
--

yaml contents go here

--